.circleci/config.yml

```diff
@@ -74,25 +73,15 @@ commands:
           steps:
             - run:
                 name: Install pnpm package manager
-                command: |
-                  corepack enable
-                  corepack prepare pnpm@latest-8 --activate
```

Comment:
I suppose we dont need this anymore because we are defining it in the package.json?

Reply:
Yep, I don't even think it was ever needed.